compass-js-frontend/src/main/webapp/model/project.js

```diff
@@ -7,11 +7,11 @@
 
 "use strict";
 
-var AmpersandModel = require("ampersand-model");
+var CompassModel = require("./compassmodel");
```

Comment:
shouldn't the file be called compass-model.js (as with other class files). Just for consistency, really.